Removed deprecated center tags

damngamerz opened the pull request:
mentors.html:removed center tag
projects.html:removed center tag
Removed both the Tags as they were deprecated
Fixes #44

sils commented:
@damngamerz did you test the changes? Can you post a screenshot of the affected pages?

damngamerz commented:
Yes !! Here you go :+1: 
![screenshot from 2017-01-30 14-51-20](https://cloud.githubusercontent.com/assets/13048113/22417740/ba07f136-e6fb-11e6-95a7-c567cc235ff4.png)
![screenshot from 2017-01-30 14-51-26](https://cloud.githubusercontent.com/assets/13048113/22417739/ba069584-e6fb-11e6-9377-90bdce06047e.png)

damngamerz commented:
@sils should i do it ?
it just needed some changes in style.css
`.profile-img`

sils commented:
@damngamerz well we can't merge it like this ;)

damngamerz commented:
@sils Here you go !!! 
found 1 more center tag to be deprecated :)
mentors.html: Removed a center Tag
style.css: Modified .profile-img
![screenshot from 2017-01-30 15-22-31](https://cloud.githubusercontent.com/assets/13048113/22418801/8a65bc6a-e700-11e6-90c4-d4e3642b212c.png)

NiklasMM commented:
ack  	e1a2fd0

sils commented on partials/tabs/mentors.html:
```diff
@@ -47,7 +47,7 @@
                 <div class="empty"></div>
                 <div class="card-action profile-data" >
-                  <center><img class="profile-img" height="100" ng-src="{{ member.github_avatar_url }}"></center>
+                  <img class="profile-img" height="100" ng-src="{{ member.github_avatar_url }}">
```
why not just apply the center-block class? Should be avail by default in bootstrap

damngamerz replied:
yup it seems to do the work!! will fix up !!! and revert the changes done in style.css

sils commented on partials/tabs/projects.html:
```diff
@@ -6,10 +6,8 @@
-          </div>
-        </center>
+        <div><a class="explore waves-effect waves-light btn">Explore</a>
+        </div>
```
the opening and closing div is redundant too btw.

damngamerz replied:
Removing this extra div tag :+1:

damngamerz commented:
@sils Tested!! with center-block seems to do the work without adding anything in style.css
Last commit logs
projects.html: Removed extra div tag.
mentors.html: added center-block to specific places.
style.css: Reverted back the changes.

sils commented:
ack https://github.com/coala/projects/pull/162/commits/918e9e8307b9f7daabcda4a8d3e59862bd285864 https://github.com/coala/projects/pull/162/commits/faef2524505593e736ee7973e261d2272e960ee1

sils commented:
rebase needed!

damngamerz commented:
@sils Found 1 more extra div tag Removed and Rebased.

hemangsk commented on partials/tabs/projects.html:
```diff
@@ -6,10 +6,7 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="text" class="validate">
       </div>
       <div class="input-field col-md-2">
```
Add a center-block here as well 👍 can you also post a screenshot of this page after changes

damngamerz replied:
@hemangsk here u go!! It remains the same though.
should i commit these changes as well ?
![screenshot from 2017-01-31 00-31-34](https://cloud.githubusercontent.com/assets/13048113/22437103/b39fe28a-e74c-11e6-97e7-b182104c639f.png)

arnab1896 commented:
ack https://github.com/coala/projects/pull/162/commits/bf4f73249b40f6faec7260c6e8e1e91bd45fbcfd

hemangsk commented:
@damngamerz Its good practice to have file names in the commit message 😃  
eg: 
```
filename: shortlog

commit body

Fixes/Closes
```
Your changes are perfect now and the commits are atomic as well 👍  👍  
Make sure to follow the commit message guidelines at [coala.io/commit](http://coala.io/commit)

damngamerz commented:
@hemangsk Done :) :+1:

hemangsk commented:
Regarding 963dfb4, why put home.html in commit message if it is not being modified at all, you can just have the commit message as Remove deprecated center tag

damngamerz commented:
@hemangsk Damn!! I missed that was copy pasting thanks for pointing. Done! :+1:

hemangsk commented:
ack 84eb429 70c9f20 74ccc23

hemangsk commented:
@rultor merge

rultor commented:
> @rultor merge

@hemangsk OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/10790-276515136)

rultor commented:
> @rultor merge

@hemangsk Done! FYI, the full log is [here](http://www.rultor.com/t/10790-276515136) (took me 1min)
